### ISSUE TITLE
Just a compliment

### ISSUE THREAD

**Issue by chris-hailstorm** [redwoodjs/redwood#164]
I hope posting a compliment here isn't a faux pas.

Redwood is conceptually strong and seems to be on a really great course.

What you said in README -- "I kept waiting for a high quality full-stack framework to arrive, but it didn't" -- so true.  There are many boilerplates and frameworks out there and most are just demos with no guidance on how to use for a custom app.  To pick on one (same could be said about many) -- the Apollo Universal Starter Kit aka AUSK looks promising but is so hard to use.

Most of the boilerplates and frameworks make the same two mistakes that Redwood has gotten right from the start -- (1) the file / project layout reflect the structure of the tools instead of the structure of the app, so the overall app flow and relationship between modules is unclear; and (2) the layers aren't generated / driven from the data model, so the dev has to manually create closely related things in the database, model, server and client -- too many moving parts.

The integrated approach you're using and especially the cells concept is strong.

Two boilerplates / frameworks I'd suggest you might use for some inspiration:

-- AUSK.  Although their overall project approach is hard to follow, the high-level modules they provide (auth, i18n, pagination, upload, others) are all desirable as reusable modules.  Maybe these would provide some ideas when you're adding similar modules to Redwood.

-- react-slingshot.  A good example on several tooling choices / configs, including Jest, SASS, PostCSS, Browsersync, Istanbul and a few other things that might be on your roadmap.

Thanks for what you're doing.  The industry needs this.

**Comment by thedavidprice** [comment]
Wind in our sails, Chris. Don't ever feel hesitant to offer encouragement + input! And please don't hesitate to keep the issues coming as they come -- we're deep in the weeds right now so real use-case feedback is valuable.

Making sure the rest of the team sees this. And also putting the projects/concepts mentioned above on the list for research.

Lastly, once you make something you're excited about, be sure to tell us about it, ok? 🚀
